.github/workflows/ci-llama-large-tests.yaml

```diff
@@ -7,6 +7,7 @@
 name: Llama Benchmarking Tests
 
 on:
+  pull_request:
   workflow_dispatch:
```

Comment:
You can also test individual workflows without creating a pull request by using workflow_dispatch: https://docs.github.com/en/actions/managing-workflow-runs-and-deployments/managing-workflow-runs/manually-running-a-workflow. Pull requests trigger other workflows (using potentially valuable CI resources) and generate notifications for everyone watching the repository.